Symfony/CS/Tests/Fixer/PSR2/BracesFixerTest.php

```diff
@@ -952,6 +952,28 @@ function ($a, $b) use ($selfName) {
         }
     );',
             ),
+            array(
+                '<?php
+class Something
+{
+public function some()
+{
+    $this->optionen = null; // default: kein Gutscheinkonto, Kreditrahmen nicht nutzen
```

Comment:
To please everyone we should probably change it to

    $this->optionen = null; // Some comment